eve/workers/docker/Dockerfile

```diff
@@ -18,8 +18,16 @@ RUN yum install -y epel-release \
     python36-pip \
     git \
     docker-ce-cli-18.09.6 \
+    which \
     && adduser -u 1042 --home /home/eve eve --groups docker \
     && chown -R eve:eve /home/eve \
     && pip3 install buildbot-worker==${BUILDBOT_VERSION}
 
+
+RUN curl -LO https://github.com/oras-project/oras/releases/download/v0.12.0/oras_0.12.0_linux_amd64.tar.gz && \
+    mkdir -p oras-install/ && \
+    tar -zxf oras_0.12.0_*.tar.gz -C oras-install/ && \
+    mv oras-install/oras /usr/local/bin/ && \
```

Comment:
you can avoid this with a single call to `tar` : e.g. by simply specifying that you want to extract only "oras".

```
tar -zxf oras_${ORAS_VERSION}_*.tar.gz -C /usr/local/bin oras
```

Reply:
sure ? i thought you must pass name filename from the archive, then rename the extracted file. I will try that

Reply:
does not work, you can't rename the file (or this might be a different command).

```
# tar cvf test.tar titi
# tar xvf test.tar -C ./bin toto                                                                                                                                
tar: toto: Not found in archive
tar: Exiting with failure status due to previous errors
```

Reply:
you are not renaming the file (`mv oras-install/oras /usr/local/bin/` is just moving the file `oras` from `oras/install` to `/usr/local/bin`) so it should work...